structure/0006-governance.md

```diff
@@ -2,53 +2,161 @@
+
+### Team Responsibilities
+
+Team responsibilities will vary and be detailed on a team-by-team basis.
+In general, a team is responsible for performing the tasks outlined in the RFC that proposed its creation.
+
+Most teams will also maintain one or more repositories on GitHub.
+The team is responsible for reviewing pull requests and ensuring that they meet The Quilt Project's goals in quality and purpose.
+Once a team has decided that a pull request is suitable for merging, a team lead will be responsible for the actual merging. Team leads can also decide to allow some or all team members to perform merges.
+
+### Creating a Team
+
+A new team can be created by passing an RFC proposing that team.
+The initial team leads must be specified in the pull request adding the RFC and must approve the RFC before it is merged.
+Once the [RFC process](0001-rfc-process.md) is concluded and the RFC is merged,
+a new team with the specified leads will be created on GitHub by the Administrative Board.
+
+The RFC proposing the new team should contain the following major sections:
+
+- **Motivation**:
+  - Why is this team needed?
+  - What problems will this team solve?
+  - What tasks will this team be responsible for?
+
+- **Explanation**:
+  - What will this team *do*?
+  - What new projects will be created that this team will be responsible for?
+  - How will each of these projects benefit the Quilt organization/community?
+  - What processes will this team follow for each of the different kinds of changes their projects may undergo?
+
+- **Drawbacks**:
+  - Is there overlap of responsibilities?
+  - Is there excessive granularity?
+
+- **Rationale and Alternatives**:
+  - Is there already a team responsible for the described tasks?
+  - If this RFC doesn't pass, which team would be responsible for the described tasks?
+
+- **Prior Art**:
+  - Are there other organizations or projects with comparable tasks?
+
+### Sub Teams
+
+Some teams may wish for more granular organization of their structure.
+In those cases, these teams may create sub teams.
+
+Sub teams behave like regular teams, except they are not required to have a team lead.
+If the sub team doesn't have its own team lead, the parent team lead's acts as its team lead.
+In general, a sub team inherits its parent's structure according to their parent's team RFC.
+
+### Triage Teams
+
+A special type of teams called Triage Teams can exist. They are here to provide support to the organization, through tasks
+such as triaging and tagging GitHub issues, quality assurance, writing regression tests, etc..
 
-The vote must be initiated with a detailed call to action, describing why the staff member feels that the person should be removed from power. The accused will have one week to submit a detailed counterargument, at which point the vote will be called.
+Members of Triage Teams aren't considered organisation members.
```

Comment:
What are the consequences of (exclusively) members of triage teams not being considered as members of the organization? Is this effectively akin to being an outside collaborator for a select number of repositories, but they are instead inside the GitHub organization (yet outside of the Quilt organization)?

There was also discussion with regards to whether a member of such a triage team would be considered a Quilt Developer, which I believe led to this additional paragraph. It then begs the question: what is the definition of a Quilt Developer?